<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1608131897875" stop="1608131973266" version="1.5.4" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Default suite : Default test</name>
    <title>Default suite : Default test</title>
    <test-cases>
        <test-case start="1608131930739" stop="1608131932633" status="passed">
            <name>TC_01_navigate_homepage_to_Doctor_login</name>
            <description type="text">Navigate_homepage_to_Doctor_loginRegister</description>
            <steps/>
            <attachments>
                <attachment title="EzHomePage" source="4da45f7f-4169-497f-8765-3cabb41c60a1-attachment.png" type="image/png"/>
            </attachments>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="sailotech.com.EzScheduler.NegativeTestPages.DoctorLoginCredentialsTest"/>
                <label name="testMethod" value="TC_01_navigate_homepage_to_Doctor_login"/>
                <label name="severity" value="normal"/>
                <label name="testId" value="TC_021"/>
                <label name="host" value="DESKTOP-A4GJKOM"/>
                <label name="thread" value="27684@DESKTOP-A4GJKOM.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1608131932825" stop="1608131932841" status="broken">
            <name>TC_02_DoctorLogin</name>
            <description type="text">Navigate_homepage_to_DoctorLogin</description>
            <failure>
                <message>FileNotFoundException: C:\Users\sukki\OneDrive\EzSchedulerscratch\EzScheduler\xls.files\DoctorIB.xlsx (The process cannot access the file because it is being used by another process)</message>
                <stack-trace>java.io.FileNotFoundException: C:\Users\sukki\OneDrive\EzSchedulerscratch\EzScheduler\xls.files\DoctorIB.xlsx (The process cannot access the file because it is being used by another process)&#13;
	at java.io.FileInputStream.open0(Native Method)&#13;
	at java.io.FileInputStream.open(FileInputStream.java:195)&#13;
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:138)&#13;
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:93)&#13;
	at sailotech.com.EzScheduler.TestImplClass.DoctorLogin.Doctorlogin(DoctorLogin.java:70)&#13;
	at sailotech.com.EzScheduler.NegativeTestPages.DoctorLoginCredentialsTest.TC_02_DoctorLogin(DoctorLoginCredentialsTest.java:73)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:498)&#13;
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)&#13;
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)&#13;
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)&#13;
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)&#13;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:648)&#13;
	at org.testng.TestRunner.run(TestRunner.java:505)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="sailotech.com.EzScheduler.NegativeTestPages.DoctorLoginCredentialsTest"/>
                <label name="testMethod" value="TC_02_DoctorLogin"/>
                <label name="severity" value="normal"/>
                <label name="testId" value="TC_22"/>
                <label name="host" value="DESKTOP-A4GJKOM"/>
                <label name="thread" value="27684@DESKTOP-A4GJKOM.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
